share/test_helpers.go

```diff
@@ -44,9 +43,10 @@ func RandEDS(t *testing.T, size int) *rsmt2d.ExtendedDataSquare {
 }
 
 // RandShares generate 'total' amount of shares filled with random data.
-func RandShares(t *testing.T, total int) []Share {
+func RandShares(t require.TestingT, total int) []Share {
 	if total&(total-1) != 0 {
```

Comment:
nit: since we are touching this function, I'd add a little comment here explaining this little trick.